app/lib/common/widgets/side_menu.dart

```diff
@@ -53,6 +53,7 @@ class SideDrawer extends StatelessWidget {
               const _GroupBudgetingItem(),
               const _SharedDocumentsItem(),
               const _GalleryItem(),
+              const _BugReportItem(),
```

Comment:
Ideally I would like this button to be placed on the screen with overlay context, so it should be accessible globally across all screens. You can use that context further to pop off once user is done reporting.

Reply:
I think this is fine fore now. In a later iteration we might overlay it everywhere or put it under the settings to make this cleaner...

Reply:
Will ignore for now.